pkg/utils/pretty/pretty.go

```diff
@@ -100,3 +102,16 @@ func ToSnakeCase(str string) string {
 func Sentence(str string) string {
 	return string(unicode.ToUpper(rune(str[0]))) + str[1:]
 }
+
+func GetWellKnownLabels() map[string]string {
```

Comment:
I'm not sure this package is really the right choice for this function - it's specifically intended to format well known labels for prometheus metrics.

Reply:
Dropped the need to move this function into the utils package